spec/models/conversation_visibilities_spec.rb

```diff
@@ -4,25 +4,16 @@
 
 require 'spec_helper'
 
-describe ConversationVisibility, :type => :model do
-  before do
-    @user1 = alice
-    @participant_ids = [@user1.contacts.first.person.id, @user1.person.id]
-
-    @create_hash = {
-      :author => @user1.person,
-      :participant_ids => @participant_ids,
-      :subject => "cool stuff",
-      :messages_attributes => [ {:author => @user1.person, :text => 'hey'} ]
-    }
-    @conversation = Conversation.create(@create_hash)
-  end
+describe ConversationVisibility, type: :model do
+  let(:user1) { alice }
+  let(:participant_ids) { [user1.contacts.first.person.id, user1.person.id] }
+  let(:create_hash) {{author: user1.person, participant_ids: participant_ids, subject: "cool stuff",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.